### DISCUSSION TITLE
How to switch to IPBanPro after installing IPBan

### DISCUSSION THREAD

**Discussion by efficiondave** [DigitalRuby/IPBan#284] [Q&A] [answered]
I installed IP Ban using the following PowerShell script:
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/DigitalRuby/IPBan/master/IPBanCore/Windows/Scripts/install_latest.ps1')) 

Now, I've installed IPBanPro. What do I need to do to remove IPBan?

**Comment by jjxtra** [comment, marked as answer]
Download this script locally: https://raw.githubusercontent.com/DigitalRuby/IPBan/master/IPBanCore/Windows/Scripts/install_latest.ps1

Then run it with a single parameter of uninstall.
